core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -891,10 +891,12 @@ private[spark] object SparkSubmitUtils {
     val localIvy = new FileSystemResolver
     val localIvyRoot = new File(ivySettings.getDefaultIvyUserDir, "local")
     localIvy.setLocal(true)
-    localIvy.setRepository(new FileRepository(localIvyRoot))
-    val ivyPattern = Seq("[organisation]", "[module]", "[revision]", "[type]s",
-      "[artifact](-[classifier]).[ext]").mkString(File.separator)
+    val ivyPattern = Seq("[organisation]", "[module]", "[revision]", "ivys",
```

Comment:
Why not add `localIvyRoot.getAbsolutePath` to this list also and remove the concatenation in the following statement?